org-jira.el

```diff
@@ -240,8 +240,8 @@ instance."
   :group 'org-jira
   :type 'boolean)
 
-(defcustom org-jira-keywords-to-jira-status-alist nil
-  "Custom alist of 'org-mode' keyword stored in car and jira status in the cdr."
+(defcustom org-jira-jira-status-to-org-keyword-alist nil
+  "Custom alist of jira status stored in car and 'org-mode' keyword stored in cdr."
   :group 'org-jira
   :type '(alist :value-type string))
```

Comment:
While at it, can you also add `:key-type string`  here so that the customize interface works properly? :)